app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="A tool to ease the distribution of very large files"
+DESCRIPTION="Jigsaw Download, or jigdo, is a tool designed to ease the \
+distribution of very large files over the internet such as CD or DVD \
+images. Its aim is to make downloading these images as easy as a click \
+on a direct download link in a browser, while avoiding all the problems \
+that server administrators could have with hosting large files."
+HOMEPAGE="https://www.einval.com/~steve/software/JTE/"
+COPYRIGHT="2004-2014 Steve McIntyre
+	2000-2010 George Danchev, Thomas Schmitt"
+LICENSE="GNU GPL v2
+	GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/download/jigit_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="459285bd743645381912301578316f2bfafb99ce1c10140a01af2b6cb011f193"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	jigit$secondaryArchSuffix = $portVersion
+	cmd:extract_data$commandSuffix = $portVersion
+	cmd:jigit$commandSuffix = $portVersion
+	cmd:jigdump$commandSuffix = $portVersion
+	cmd:jigsum$commandSuffix = $portVersion
+	cmd:jigit_mkimage$commandSuffix = $portVersion
+	cmd:mkjigsnap$commandSuffix = $portVersion
+	cmd:rsyncsum$commandSuffix = $portVersion
+	lib:libjte$secondaryArchSuffix = $portVersion
```

Comment:
The actual version is "1.0.0" for libjte so suggestion here and in devel with:
```diff
-	lib:libjte$secondaryArchSuffix = $portVersion
+	lib:libjte$secondaryArchSuffix = 1.0.0 compat >= 1
```
Or you could use libVersion(Compat) here :)